### ISSUE TITLE
Why do jungles make you hot

### ISSUE THREAD

**Issue by Shadie116** [jasonrohrer/OneLife#996]
Last time I walked into a jungle I don't recall it being hot. Also why don't they got no mosquitos in the swamp? Mosquitos are just bratty bastards, you'd expect them to be in the place that has the most calm water ponds.

**Comment by Shadie116** [comment]
Mi persona es cocido under the arborol.

**Comment by is52hertz** [comment]
Yes, this game - you can see seals, penguins, squirrels, glaciers and the underlying ocean all at once in the snow.
In reality, none of them can appear together.

**Comment by Shadie116** [comment]
I know, terrifying. We are surrounded by Canada 😨

**Comment by is52hertz** [comment]
OHOL should be supported
https://www.curseforge.com/minecraft/mc-mods/terraforged
Haha, that's not going to happen.
But the terrain in this mod is really nice.

**Comment by DougLefelhocz** [comment]
Jungles' temperature has worked like this since the temperature overhaul in February of 2019.  The game has had a mechanism called "temperature shock" which instantly changes temperature and has since 2019.  Mosquitoes got put there, so that no one family could successfully settle in a jungle, back before race restrictions existed.  Skin color is not an adaptation to the sun in this game.  It's for different character models, as not only does different skin colors predate race restrictions, it does exist when a server has less than 30/40 players on it.

**Comment by Shadie116** [comment]
well than you can untemperature shock it. Also look at this thing I found in the congo country
![image](https://github.com/jasonrohrer/OneLife/assets/126846097/3d43724b-7dac-43cb-82ae-5a38e0693b34)

**Comment by Shadie116** [comment]
They seem to be covered in earth fungus and they're ok
